### PR TITLE
Apply changes suggested by dfmt

### PR DESCRIPTION
**Pull request by yebblies**

### CODE REVIEW

**Comment by ZombineDev** [comment]
Can you apply `dfmt` on top of the first three commits of Kenji's [PR](https://github.com/D-Programming-Language/dmd/pull/5239/commits)? Those style changes should not be affected by `dfmt` anyway.
After this is done future PRs to DMD can have far less formatting fixes.

**Comment by yebblies** [comment]
If I did that, who would merge them?

File: src/cppmangle.d [hunk 1]
```diff
@@ -1004,15 +1004,15 @@ else static if (TARGET_WINDOS)
...
-                    case Tint64:
-                    case Tuns64:
-                    case Tint128:
-                    case Tuns128:
-                    case Tfloat80:
-                    case Twchar:
-                        if (checkTypeSaved(type))
-                            return;
-                    default:
+                case Tint64:
```
Comment (AndrejMitrovic) [inline src/cppmangle.d]: Btw this is a weird formatting style. Is dfmt hardcoded or is it configurable?

Comment (yebblies) [inline src/cppmangle.d]: It's the style used throughout dmd.

Comment (AndrejMitrovic) [inline src/cppmangle.d]: Aww. I prefer consistency over my personal taste so ok then. :)

**Comment by AndrejMitrovic** [comment]
> Can you apply dfmt on top of the first three commits of Kenji's PR?

I'm not sure what is being asked here, aren't those commits already in master now since that PR was merged?

I'd just merge this PR as is. @yebblies let me know if you agree.

**Comment by yebblies** [comment]
> I'm not sure what is being asked here, aren't those commits already in master now since that PR was merged?

It wasn't merged.

> I'd just merge this PR as is. @yebblies let me know if you agree.

Yes please.

**Comment by ibuclaw** [comment]
If this is all that was fixed running dfmt on the code, could we add a new travis test to check that all new PRs don't divulge from the standard format?

**Comment by yebblies** [comment]
> If this is all that was fixed running dfmt on the code, could we add a new travis test to check that all new PRs don't divulge from the standard format?

No, it also undid 10000 lines of Kenji's formatting changes.  This is just the lines that I felt corresponded to accidental changes.

**Comment by MartinNowak** [comment]
The way I see it is that regularly fixing the style here and there causes constant code churn.
I think we should sort of #5217 (which might include a few dfmt fixes), integrate dfmt in our build system, and then enforce it for any PR.
If we wanted to do it incrementally we could "whitelist" one file after another, but I'd be in favor of doing all at once.

Action Points:
- [ ] file and/or tag dfmt issues blocking adoption (weird enough you guys want aligned comments)
- [ ] work w/ @Hackerpilot on fixing those
- [ ] integrate dfmt w/ our build infrastructure
- [ ] reformat all dmd code
- [ ] enforce code style in tester

**Comment by yebblies** [comment]
> The way I see it is that regularly fixing the style here and there causes constant code churn.

Most of this is errors that I accidentally introduced when manually porting to D.  It should be fixed before doing the rest.

**Comment by MartinNowak** [comment]
Then please rebased and we're good to go.

**Comment by MartinNowak** [comment]
Auto-merge toggled on

**Comment by yebblies** [comment]
I rebased this and forgot about it.  Thanks Martin.

**Pull request merged**